test/e2e/app-dir/navigation/navigation.test.ts

```diff
@@ -80,11 +80,12 @@ createNextDescribe(
             // App Router doesn't re-render on initial load (the params are baked
             // server side). In development, effects will render twice.
 
-            // experimental react is having issues with this use effect
-            // @acdlite will take a look
-            // TODO: remove this PPR cond after react fixes the issue in experimental build.
-            waitForNEffects:
-              isNextDev && !process.env.__NEXT_EXPERIMENTAL_PPR ? 2 : 1,
+            // BACKPORT NOTE:
+            // newer StrictMode does not double invoke effects during hydration
+            // https://github.com/facebook/react/pull/28951
+            // but the backport branch is still on older react
+            // waitForNEffects: 1,
+            waitForNEffects: isNextDev ? 2 : 1,
```

Comment:
@ztanner i adjusted this because `__NEXT_EXPERIMENTAL_PPR=true` still gave me a 2 here. any idea why this'd change here? that's a bit suspicious